doc: document that server.address() can return null (#32519)
nodejs/node: doc/api/net.md

});
```

> If you call `server.address()` before the `'listening'` event has been

Reviewer: To be pedantic `null` is returned also after `close()`.

Author: Fixed 👍

> or after `server.close()` has been called, `null` will be returned.

Reviewer 2: ```suggestion
If `server.address() is called before the `'listening'` event has been emitted,
or after `server.close()` has been called, `null` will be returned.
```

Reviewer 3: +1 on removing _you_, but how about we replace it with active voice rather than passive voice? And keep it nice and short?

```suggestion
`server.address()` returns `null` before the `'listening'` event or after
calling `server.close()`.
```

Author: @Trott I find your version a bit harder for my brain to parse, but maybe it's just me. But if I edit it slightly it's easier for me:

```suggestion
`server.address()` returns `null` before the `'listening'` event has been emitted
or after calling `server.close()`.
```

Reviewer 4: @watson To me that sounds good, do you want to apply it?
